vipps-recurring-api.md

```diff
@@ -635,28 +668,41 @@ a description with follow format `{agreement.ProductName} - {charge.description}
 ### Charge times
 
 Charge _attempts_ are primarily made two times during the day: 07:00 and 15:00 UTC.
+Vipps may do extra attempts and/or change this without notice.
 The processing of charges typically takes around one hour, however this varies and we do not guarantee any time.
 This is the same both for our production and test environment.
 Subsequent attempts are made according to the `retryDays` specified.
 
-**Note:** Payments _might_ get processed any time during the day (07:00 UTC - 23:59 UTC) due to special circumstances requiring it.
+**Note:** Payments _might_ get processed any time during the day
+(07:00 UTC - 23:59 UTC) due to special circumstances requiring it.
 
 **Note:** Since payments _can_ be processed any time (07:00UTC - 23:59 UTC) it is advisable to fetch the charge at/after 00:00 UTC the day after the last retry day to be sure you get the last status.
 
-### Charge retries
+When a charge has reached its `due` date, the status of the charge will be
+`PROCESSING` until the charge is successful, for as long as the merchant has
+specified with `retryDays`.
+
+**Important:** Vipps does not "leak" the customers's information about insufficient funds,
+blocked cards, etc. Users are informed about all such problems in Vipps,
+which is the only place they can be corrected. The merchant's customer service
+should always ask the user to check in Vipps if a charge has failed.
 
-Vipps will retry the charge for the number of days specified in `retryDays`. The maximum number of retryDays is currently set to 14 days
+### Charge retries
 
-So if `retryDays=2` that would mean a maximum of 6 retries:
-Two on the initial day, and two each for the subsequent days.
+Vipps will retry the charge for the number of days specified in `retryDays`.
+The maximum number of `retryDays` is 14.
 
-If `retryDays=0` it will try two times on the initial day.
+The status of a charge will be `PROCESSING` while Vipps is taking care of business,
+from the `due` date until the charge has succeeded, or until the
+`retryDays` have passed without a successful charge.
+The final status will be `CHARGED` or `FAILED`.
+See the API documentation for more details.
 
-Be aware that if you check the status of the charge within the retry period, it
-might have status `FAILED`, also after the first attempt on the first and only day.
```

Comment:
I see you're planning on removing this text from the documentation. Does it mean the discussion in #143 can be closed as well? In general, this fragment made some of us think that the charge can land in the `FAILED` state after the first attempt on the first and only day but be later on changed to `CHARGED` if the second attempt on the first and only day succeeded. 

Followed by `If a charge fails, you will get information about the reason: Charge failure reasons.` it actually sounded like we must check whether the reason is disclosed to ensure the charge is beyond repair.

Reply:
Yes, the "temporary" fail status has caused a lot of confusion.

We will have the experimental failureReason and failureDescription for a little while, but will provide an event log as soon as possible. 

We will also inform about attempted captures where the capture amount is higher than the user has _so far_ approved. 

In general: We want to take care of everything, we want to "leak" only the necessary details, and we are working har don an event log - which should match event logs for others APIs too.

See the newsletter for updates (when they come): https://github.com/vippsas/vipps-developers/tree/master/newsletters